src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -192,4 +192,25 @@ public static Status parseStatus(String status) throws ParseException {
         }
         return new Status(trimmedStatus);
     }
+
+    /**
+     * Parses a {@code String} delivery attribute to ensure it is a valid delivery attribute.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code String} delivery attribute is invalid.
+     */
+    public static String parseAttribute(String attribute) throws ParseException {
+        requireNonNull(attribute);
+        String trimmedAttribute = attribute.trim();
```

Comment:
Might want to consider the case of having differing cases (e.g. `ADDRESS` instead of `address`) and whether the attribute should be case-insensitive

Reply:
This has been fixed!